README.md

```diff
@@ -1,36 +1,60 @@
 # ContextData
-Context data object specifications, schemas, and examples
+Context data object specifications, schemas, and examples:
 
 * Extending APIs from one App to another is powerful...
 * However, it requires building to a specific API ahead of time
 * Standard context and intent definitions let us create workflows on the fly
 
-FDC3 Context Data defines a standard way to pass common identifiers between apps to create a seamless workflow.  FDC3 Context Data is not a symbology solution and is not focused on modeling financial objects.  The focus is on providing a standard envelope and a standard set of identifiers that can be used to set a lowest common denominator for interop.
+FDC3 Context Data defines a standard way to pass common identifiers and data between apps to create a seamless workflow.  FDC3 Context Data is not a symbology solution and is not specifically focused on modeling financial objects.  The focus is on providing a standard payload structure that can be used to establish a lowest common denominator for interoperability.
 
-## Context Envelope Structure
-```javascript
-{
-object: (string) //object type
-definition: (string) //url
-version: (string) //version # (semver)
-data: (array) //the context payload
+Context objects are used when [declaring]() and [raising]() intents, and when [broadcasting]() context to other applications.
+
+## Context Object
+
+Context can be summarised as:
+* Having a unique _$type_ identifier, used for routing.
+* Optionally providing a name.
+* Optionally providing a map of equivalent identifiers.
+* Any other properties or metadata.
+```typescript
+interface Context {
+    $type: string;
```

Comment:
Since `$type` is what is going to be used to figure out the intents, if an app needs to specify that they only accepts a particular symbology (e.g. `com.factset.symbology.entity`) they would need to create a new type and register that. Right?

```
interface FactSetEntityID extends Context {
    $type: 'com.factset.symbology.entity',
    name: string;
    id: {
        value: string;  // in most cases like these, "id" will be an object with single key/value pair.
    }
}
```

Reply:
That depends on the scope of resolution; if this type is to be used only internally by a platform (e.g Factset), then resolution and routing can be decided by the platform without public registering.

Reply:
@vinodmehta I think you misunderstand. This proposal is not about standardising identifiers, but about standardising representations of entities. E.g. you would standardise a security, instrument, contact or position, or even an RFQ or butterfly trade.

Identifiers for those entities are more often than not already standardised, e.g. ISIN, and we just use them in the representations. Even something like the FactSet entity id is not our job to standardise or represent, that is FactSet's job.

A security would just reference the FactSet identifier, as in the example included in the PR:

```json
{
    "$type" : "fdc3.instrument",
    "name" : "Apple",
    "id" : 
    {  
        "ticker" : "aapl",
        "ISIN" : "US0378331005",
        "CUSIP" : "037833100",
        "FIGI" : "BBG000B9XRY4",
        "com.factset.symbology.entity": "000C7F-E"
    }
}
```
Whether we include the FactSet identifier in the list of well-known identifiers for an instrument is open for discussion. It could be "extra", or potentially form part of the standard definition, e.g.:

```typescript
interface Instrument extends Context {
    $type: 'fdc3.instrument';
    name: string;
    id: {
        ticker?: string;
        ISIN?: string;
        CUSIP?: string;
        SEDOL?: string;
        RIC?: string;
        BBG?: string;
        PERMID?: string;
        FIGI?: string;
        'com.factset.symbology.entity'?: string;
    }
} 
```
Note that the majority of the identifiers are optional here, they can all be excluded, or all included, and based on how the `Context` interface defines the id map, you can include any extra ones you want outside of this list of well-known ones.

This is an important part of the use case: if you are trying to pass reference data, include as many as possible identifiers to allow the target application to identify the context.